### ISSUE TITLE
Easy conversion from "New"

### ISSUE THREAD

**Issue by Jeppedy** [pucherot/Pi.Alert#226]
Is there any way to quickly convert the first set of devices as "not new"?

**Comment by leiweibau** [comment]
During the installation you will be asked if the devices found during the installation should be marked as "new". However, the problem here is that you don't actually know what this means yet.

In my fork (https://github.com/leiweibau/Pi.Alert) I have integrated a bulk editor, which allows quick editing. https://github.com/leiweibau/Pi.Alert/blob/main/docs/BULKEDITOR.md

**Comment by Jeppedy** [comment]
Thanks. I'm using a docker image, which makes using an individual fork a
bit more work, but I may build my own using your version.  Thanks again!

**Comment by jokob-sk** [comment]
If you are using [my fork](https://github.com/jokob-sk/Pi.Alert) you might use the Maintenance > Backup restore > CSV Export > Mass replace > CSV import workaround.

**Comment by Jeppedy** [comment]
Yep, that's the one!  Where is the "mass replace" function?  You mean like
just an Excel find/replace type of thing?

**Comment by jokob-sk** [comment]
yes, but I recommend software that doesn't change line endings, such as notepad++
